doc/source/WorkingWithThePythonScriptingAPI/ThePythonEditor/index.md

```diff
@@ -0,0 +1,72 @@
+```eval_rst
+.. role:: raw-html(raw)
+    :format: html
+```
+
+# The Python Editor #
+
+The _Python Editor_ is a type of editor in the main Gaffer application that lets you build and modify the node graph, test expressions, and inspect data by inputting and executing Python code. In the default layout, it resides in the bottom-right panel, in a tab next to the _Hierarchy View_.
+
+The _Python Editor_ is split into two areas. The bottom-half is the input field. The top-half is the output. The input field of the _Python Editor_ functions like a basic plain text editor. You can type, select, cut, copy, paste, and execute code. 
+
+![A Python Editor with its input and the output field](images/interfacePythonEditorOutput.png "A Python Editor with its input and the output field")
```

Comment:
It might just be my local build, but the caption seems to be inline with the image:
![image](https://user-images.githubusercontent.com/896779/58035806-7941bf80-7b21-11e9-96a9-7af936e5bbd9.png)

Reply:
Which version of ReadTheDocs/Recommonmark did you build with? This looks like [a certain bug](https://github.com/rtfd/recommonmark/issues/88) in more recent versions of Recommonmark.

Reply:
Looks like:
```
recommonmark                           0.5.0   
Sphinx                                 1.8.5   
sphinx-rtd-theme                       0.4.3 
```  
Ill try the container build and see how it looks. Thanks!